oneapp-server/types/Application.js

```diff
@@ -0,0 +1,25 @@
+const { gql } = require('apollo-server-express');
+
+const typeDef = gql`
+  extend type Query {
+    "Safety net benefits application for SNAP, TANF, GA"
+    application: Application
+  }
+
+  type Application {
+    "The id of the application. This will be the same as the user id."
+    APPLICATION_NUMBER: ID
```

Comment:
This does not need to be present in the application model, but I figured it may be helpful for the front-end to have some type of id to key off of.